_guide/_pages/development-environments.md

```diff
@@ -64,7 +66,8 @@ before_script:
 
 Now each set of scripts are much easier run in other environments: locally, in a different CI environment, etc. Moreover, think of the YAML keys as annotations for the scripts.
 
-
 **Reproducible**
 
 A good development environment should be reproducible across different computers, platforms, and environments. Reproducibility helps ensure that bugs are not idiosyncratic to any one person's bespoke computing environment--rather they are intrinsic to the repository itself such that time can be spent debugging the repository code and not the environment on a person's computer. Pinning dependencies such as language runtimes and databases to specific versions is a great way to help achieve this.
+
+[Docker]({{site.baseurl}}/project-setup/docker/) is a good way to achieve this.
```

Comment:
```suggestion
```

Reply:
Can you say more about this? Use of Docker helps with both ease of setup and reproducibility.

Reply:
Duplicates line 19?